File: 0217-contains-duplicate/0217-contains-duplicate.cpp
class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        sort(nums.begin(), nums.end());

        int next = 1;
        for (int i = 0; i < nums.size()-1; i++) {
            if (nums[i] == nums[next]) return true;
            next++;
        }

        return false;
    }
};

File: 0217-contains-duplicate/0217-contains-duplicate.go
func containsDuplicate(nums []int) bool {
    var counts map[int]int
    counts = map[int]int{}

    for _, num := range nums {
        if counts[num] > 0 {
            return true
        }

        counts[num] = counts[num] + 1
    }

    return false
}